dev.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14.15.1-alpine
```

Comment:
Just realized this is a different node version for this container. Below behavior persists even in node 12.x.x